README.md

```diff
@@ -35,6 +35,11 @@
 ```sh
 npm install
 ```
+#### Run the following command to generate the `supportedBrowsers.ts` file:
+
+```bash
+npm run supported-browsers
+```
```

Comment:
Can we add a short explanation as to why this extra step is needed and what it does?

Reply:
npm run supported-browsers ensures the project’s code is compatible with the specified browsers in the browserslist config. It adds necessary polyfills and transforms modern code to work across supported browsers.

**_is it enough ?_**

Reply:
where should i add these line ?